rero_ils/modules/documents/api.py

```diff
@@ -162,3 +156,18 @@ def dumps(self, **kwargs):
         for edition in editions:
             edition['_text'] = edition_format_text(edition)
         return dump
+
+    def index_persons(self, bulk=False):
+        """Index all persons attacht."""
```

Comment:
I suggest `all attached persons` in `def index_persons` comment.

Reply:
ok